docs/intro/tutorial01.txt

```diff
@@ -173,13 +173,14 @@ It worked!
 
         $ python manage.py runserver 8080
 
-    If you want to change the server's IP, pass it along with the port. So to
-    listen on all public IPs (useful if you want to show off your work on other
-    computers on your network), use:
+    If you want to change the server's IP, pass it along with the port. For
+    example, to listen on all available public IPs (which is useful if you are 
+    running Vagrant or want to show off your work on other computers on the 
+    network), use:
 
     .. code-block:: console
 
-        $ python manage.py runserver 0.0.0.0:8000
+        $ python manage.py runserver 0:8000
```

Comment:
I wonder if a sentence of explanation indicating that 0 is short for "0.0.0.0" would be helpful. I wasn't familiar with this until recently.